caom2pipe/tests/data/test_cache.yml

```diff
@@ -0,0 +1,14 @@
+value_repair:
+  observation.type:
+    dark: DARK
+    Dark: DARK
+  observation.proposal.pi_name:
+    jjk: JJ Kavelaars
+  plane.meta_release:
+    any: 2000-01-01T00:00:00.000
+  chunk.position.axis.function.ref_coord.coord1.pix:
+    512.579594886106: 512.57959987654321
+  artifact.uri:
+    gemini:GEM: cadc:GEMINI*
+  part.product_type:
+    CALIBRATION: none
```

Comment:
I'm wondering if it wouldn't be more powerful if you allow `REGEX` here. I think that with the exception of `none`, all these cases can be expressed with `REGEX`.

Reply:
Done.